src/components/MenuPanel/MenuPanel.js

```diff
@@ -182,8 +182,8 @@ const Main = styled.div`
   display: flex;
   flex-direction: column;
   width: 220px;
-  height: 100%;
-  flex-shrink: 1;
+  flex-grow: 0;
+  flex-shrink: 0;
```

Comment:
I think `flex-grow` and `flex-shrink` default to 0, so this might not be necessary, but we do use them separately in a lot of places.

Reply:
[Nevermind me, I am clearly wrong](https://www.w3.org/TR/css-flexbox-1/#flex-flex-grow-factor).